src/Executor/ReferenceExecutor.php

```diff
@@ -576,11 +585,13 @@ function ($results, $responseName) use ($contextValue, $path, $parentType, $root
      * by calling its resolve function, then calls completeValue to complete promises,
      * serialize scalars, or execute the sub-selection-set for objects.
      *
-     * @param mixed                       $rootValue
-     * @param array<int, string|int>      $path
-     * @param mixed                       $contextValue
+     * @param mixed                  $rootValue
+     * @param array<int, string|int> $path
+     * @param array<int, string|int> $unaliasedPath
+     * @param mixed                  $contextValue
      *
-     * @phpstan-param Path                $path
+     * @phpstan-param Path           $path
+     * @phpstan-param Path           $unaliasedPath
```

Comment:
It would be great if phpdoc_align was set to left, to not cause all these changes.

https://cs.symfony.com/doc/rules/phpdoc/phpdoc_align.html

Reply:
Can you create a separate pull request for that?

Reply:
Undid any unrelated changes.